src/dmd/dtoh.d

```diff
@@ -171,6 +173,33 @@ private bool isFrontendModule(ASTCodegen.Module m)
             (m.parent.parent.ident == DMDModule.dmd && !m.parent.parent.parent));
 }
 
+private string translateBasicType(ubyte ty)
```

Comment:
I might be having dejavu, but isn't this change already in master? Perhaps a rebase is in order so that @TurkeyMan's changes aren't mixed up with this.

Reply:
Ummm, yeah not sure why this is pulling my patch in too?
Can we make a call on my one before merging this?

Reply:
Yes.  Will be done before PR is done.